base/server/python/pki/server/cli/plugin.py

```diff
@@ -0,0 +1,818 @@
+
+        instance_name = 'pki-tomcat'
+        for o, a in opts:
+            if o in ('-i', '--instance'):
+                instance_name = a
+
+            elif o == '--help':
+                self.print_help()
+                sys.exit()
+            else:
+                print('ERROR: unknown option ' + o)
+                self.print_help()
+                sys.exit(1)
+        plugin_type = args[0].strip()
+        if not plugin_type:
+            print("ERROR: Plugin type required")
+            sys.exit(1)
+
+        instance = pki.server.PKIInstance(instance_name)
+        if not instance.is_valid():
+            print('ERROR: Invalid instance %s.' % instance_name)
+            sys.exit(1)
+
+        instance.load()
+
+        subsystem_name = self.parent.parent.parent.name
+        subsystem = instance.get_subsystem(subsystem_name)
+        deleted = False
+        for key in list(subsystem.config.keys()):
+            if key.startswith('auths.impl.{}.'.format(plugin_type)):
+                del subsystem.config[key]
+                deleted = True
+        subsystem.save()
+        if deleted:
+            print("Auth plugin manager {} deleted.".format(plugin_type))
+        else:
+            print("ERROR: Auth plugin manager {} not found.".format(plugin_type))
+            sys.exit(1)
+
+
+class AuthPluginManagerFindCLI(pki.cli.CLI):
+    def __init__(self, parent):
+        super(AuthPluginManagerFindCLI, self).__init__(
+            'find', 'Find auth plugin manager'
+        )
+        self.parent = parent
+
+    def print_help(self):
+        print("Usages: pki-server %s-auth-plugin-manager-find "
+              % self.parent.parent.parent.name)
+        print()
+        print('  -i, --instance <instance ID>       Instance ID (default: pki-tomcat).')
+        print('      --help                         Show help message.')
+        print()
+
+    def execute(self, argv):
+        try:
+            opts, _ = getopt.gnu_getopt(argv, 'i:', [
+                'instance=', 'help'])
+
+        except getopt.GetoptError as e:
+            print('ERROR: ' + str(e))
+            self.print_help()
+            sys.exit(1)
+
+        instance_name = 'pki-tomcat'
+        for o, a in opts:
+            if o in ('-i', '--instance'):
+                instance_name = a
+
+            elif o == '--help':
+                self.print_help()
+                sys.exit()
+            else:
+                print('ERROR: unknown option ' + o)
+                self.print_help()
+                sys.exit(1)
+
+        instance = pki.server.PKIInstance(instance_name)
+        if not instance.is_valid():
+            print('ERROR: Invalid instance %s.' % instance_name)
+            sys.exit(1)
+
+        instance.load()
+
+        subsystem_name = self.parent.parent.parent.name
+        subsystem = instance.get_subsystem(subsystem_name)
+
+        print("  Configured Plugin Managers.")
+        print("  ===========================")
```

Comment:
Let's remove this header since the output should be clear from the command being executed.